nasademo/src/main/java/com/example/nasademo/db/Camera.java

```diff
@@ -0,0 +1,33 @@
+package com.example.nasademo.db;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import java.time.LocalDate;
+
+@Getter
+@Setter
+@AllArgsConstructor
+@NoArgsConstructor
+@Entity
+@Table(name = "cameras")
+public class Camera {
```

Comment:
I noticed you have 2 classes of Camera. Let's rename this class to `CameraEntity`. The same for the `Picture` entity